tests/metadata-service-tests/test/testkit/k8sresources_client.go

```diff
@@ -1,8 +1,8 @@
 package testkit
 
 import (
-	"github.com/kyma-project/kyma/components/gateway/pkg/apis/istio/v1alpha2"
-	istioclient "github.com/kyma-project/kyma/components/gateway/pkg/client/clientset/versioned"
+	"github.com/kyma-project/kyma/components/application-connector/pkg/apis/istio/v1alpha2"
```

Comment:
The path on this branch is `.../metadata-service/pkg/...`, please update this after the changes are merged. (same case for next import)